kitchen-sink-example.yaml

```diff
@@ -1,11 +1,58 @@
 # The file format version
 file_format: "0.1"
 
+# Configure general attribute limits. See also tracer_provider.span_limits, logger_provider.log_record_limits.
+attribute_limits:
+  # Set the max attribute value size.
+  #
+  # Environment variable: OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT
+  attribute_value_length_limit: 4096
+  # Set the max attribute count.
+  #
+  # Environment variable: OTEL_ATTRIBUTE_COUNT_LIMIT
+  attribute_count_limit: 128
+
 # Configure logger provider.
-logger_provider: {}
+logger_provider:
+  # Configure the log record limits. See also attribute_limits.
+  log_record_limits:
+    # Set the max log record attribute value size. Overrides attribute_limits.attribute_value_length_limit.
+    #
+    # Environment variable: OTEL_LOGRECORD_ATTRIBUTE_VALUE_LENGTH_LIMIT
+    attribute_value_length_limit: 4096
```

Comment:
since these are already under `_limits`, does it make sense to remove the `_limit` suffix?

Reply:
I don't know why but also think I prefer having the `_limit` suffix.

Reply:
i'm ok with keeping the suffix for now, it can always be revisited later